blocks/header/header.css

```diff
@@ -38,24 +42,30 @@ header nav .text {
   white-space: nowrap;
 }
 
+/* Table view (--udexGridSMinWidth) */
 @media (width >= 640px) {
   header nav {
     padding: 0 48px;
   }
 }
 
+/* Desktop view (--udexGridMMinWidth) */
 @media (width >= 980px) {
   header nav {
     padding: 0 72px;
   }
 }
 
+/* XL Desktop view (--udexGridXLMinWidth) */
 @media (width >= 1600px) {
   header nav {
     padding: 0 246px;
   }
 }
 
+/* #todo: it appears this media query can be collapsed with the previous one */
```

Comment:
I see multiple media queries with the same breakpoints. Should these be consolidated under single media query breakpoint? I find it to easier to manage it that way in the longer run. BTW, these multiple breakpoint pre-exist and not something being added via this PR so can be addressed separately.

Reply:
@sdmcraft It depends on organisation. A large dev community prefers keeping separate media query for separate rule. This way in one CSS it is clear to understand how a particular rule applies for every breakpoint. 
Once such reference:
[CSS Organisation](https://stackoverflow.com/questions/11605761/how-should-css3-media-queries-be-managed)